tests/unit/components/Editor/ItemEditor.spec.js

```diff
@@ -0,0 +1,30 @@
+import { mount } from "@vue/test-utils";
+import ItemEditor from "@/components/Editor/ItemEditor";
+import Tooltip from "primevue/tooltip";
+import VueClickAway from "vue3-click-away";
```

Comment:
We're importing clickaway and tooltip multiple times in multiple component tests. Wouldn't it make sense to just import it globally in jest config?